### ISSUE TITLE
Instructions?

### ISSUE THREAD

**Issue by sirensynapse** [Seadox/Bumble-likes-viewer#9]
Hi, your instructions are as clear as mud. At least to a noob.
What is and where is a "project folder"?
WTF does this mean?
Install requirements:
pip install -r requirements.txt
When I try this bit (sure it's fucked since I didn't do the above steps yet), windows just asks me what program I use to open it.  
Run:

python bumble.py

Can you just type your instructions in plain english for normal people, not hackers and programmers who are utterly clueless?
Who do you think is using this, actually? Hint: noobs on dating apps. 
Why is everyone on github making their offerings to utterly fucking obtuse for normal people to use? AHHHHHHHHHHHHHHHHH!!!
Thanks

**Comment by DarkVillager** [comment]
I'm working on a different version where you don't need anything other than a browser. ( https://github.com/DarkVillager/bumble-likes-viewer-browser )
